client/src/client.rs

```diff
@@ -338,12 +338,13 @@ impl_query_output! {
 }
 
 /// Iroha client
-#[derive(Clone, DebugCustom, Display)]
+#[derive(Clone, DebugCustom, Display, getset::Getters)]
```

Comment:
Why do we need getters here?

Reply:
As you can see in your another comment, I use `client.account_id()`. I think it's more readable and makes more sence than `alice@wonderland` here

Reply:
I think it's okay for `Client` to have getters for fields it's configured with

Reply:
and did you consider making the fields public?

Reply:
Hm, no

Reply:
Although it's a crate which will be used by external clients a lot. So I think having methods instead of direct field access is better for future API changes

Reply:
> Although it's a crate which will be used by external clients a lot. So I think having methods instead of direct field access is better for future API changes

I disagree. What is the difference if you use field vs method access?

Reply:
To me it sounds natural to have `pub` fields only in [POD types](https://en.wikipedia.org/wiki/Passive_data_structure). For complex types with complex logic I'd prefer getters.

Objectively having getters allows us to change structure fields if future while still mimicing the same API for the end users.

Reply:
It creates opaqueness which is not bad for a crate which will be used by external people

Reply:
how is this opaqueness useful? it serves no tangible purpose. If you can tell me how method provides you with more opaqueness than `pub` fields I will accept this

Reply:
Here it is:

> Objectively having getters allows us to change structure fields if future while still mimicing the same API for the end users.

Actually I don't mind making fields public